code/development-env/README.md

```diff
@@ -1,113 +1,102 @@
-  * You want `/usr/local/bin/kubectl` to be the one from Homebrew - to do this, run `brew link --overwrite kubernetes-cli`
-* [Install VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-* [Allow system software from Oracle](https://stackoverflow.com/questions/52277019/how-to-fix-vm-issue-with-minikube-start)
-* [Install minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
-* Install yarn: `brew install yarn`
-* Install jq: `brew install jq`
-* (Recommended) Install dnsmasq: `brew install dnsmasq`
-
-### Set-up minikube
-
-* Start minikube (you can ensure virtual box uses the same IP address by following [these instructions](#clearing-minikube-ip-address-when-creating-new-cluster-macos))
-  * `minikube start`
-  * Or give the minikube cluster a different name using the `-p` flag e.g. `minikube -p datalabs start` (note that you will need to us the `-p` flag to refer to this instance of minikube in subsequent commands e.g. `minikube -p datalabs stop`).
-* (Recommended) [Install `kubectx` and `kubens`](https://github.com/ahmetb/kubectx)
-  for easy context and namespace switching.
-* Create a Gluster and NFS storage class by running `kubectl apply -f ./config/manifests/minikube-storage.yml`
-* Create compute submission cluster role by running `kubectl apply -f ./config/manifests/minikube-compute-submission-role.yml`
-* (Other potentially useful manifests can be found in `./config/manifests/`)
-
-#### Clearing Minikube IP address when creating new cluster (MacOS)
-
-VirtualBox will retain leases for local IP address, when creating a new cluster
-a new IP address will be assigned. These can be cleared by deleting VirtualBox
-DHCP leases (`rm ~/Library/VirtualBox/HostInterfaceNetworking-vboxnet0-Dhcpd.*`)
-
-The current IP for minikube can be found using `minikube ip`.
-
-### Install NodeJS packages
-
-* Run `yarn` in each of the following directories:
-  * `code`
-  * `code/auth-service`
-  * `code/infrastructure-api`
-
-Running the `yarn` command within the `code/` directory will install all the
-packages required for services within the workspace (these services within the
-`code/workspaces` directory; Web-App, Client-API, etc).
+Unless stated otherwise, the following instructions are for MacOS Catalina.
 
-Git-hooks (via husky) will enable when running the install step above. This will
-enforce linting rules for any files being staged. Any errors highlighted will
-need to be address before the git commit will be permitted. This rule checking
-may be disabled using the `--no-verify` flag with `git commit`, this is not
-recommended as linting error will still be caught with the CI server.
+## Installations
+
+### Essential
+
+* [brew](https://brew.sh/)
+* [Node Version Manager](https://github.com/nvm-sh/nvm)
+* Node.js v12/Erbium: `nvm install lts/erbium`
+* [Yarn](https://yarnpkg.com/getting-started/install)
+* [Docker](https://docs.docker.com/get-docker/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
```

Comment:
Doesn't compose come bundled with Docker for Desktop that would be used if working on Mac?

Reply:
Yes, I'll make that explicit.